src/main/java/com/rtg/vcf/eval/VcfEvalCli.java

```diff
@@ -400,29 +403,84 @@ protected VcfEvalParams makeParams() throws IOException {
    * @throws IOException if a region based filter is specified but cannot be loaded
    */
   public static Set<RocFilter> getRocFilters(CFlags flags, List<RocFilter> fallbackRocFilters) throws IOException {
-    final Set<RocFilter> rocFilters = new LinkedHashSet<>(Collections.singletonList(RocFilter.ALL));  // We require the ALL entry for aggregate stats
-    if (flags.isSet(VcfEvalCli.ROC_SUBSET)) {
-      final List<?> values = flags.getValues(VcfEvalCli.ROC_SUBSET);
-      for (Object o : values) {
-        rocFilters.add(((RocFilterProxy) o).filter());
-      }
-    }
+    final LinkedHashMap<String, Set<RocFilter>> rocFiltersTypeMap = new LinkedHashMap<>();
+
+    final Set<RocFilter> rocExprFilters = new LinkedHashSet<>();
     if (flags.isSet(ROC_EXPR)) {
       final List<?> values = flags.getValues(VcfEvalCli.ROC_EXPR);
       for (Object o : values) {
         final String[] e = StringUtils.split((String) o, '=', 2);
         assert e.length == 2;
-        rocFilters.add(new ExpressionRocFilter(e[0], e[1]));
+        rocExprFilters.add(new ExpressionRocFilter(e[0], e[1]));
       }
     }
+    rocFiltersTypeMap.put("rocExprFilters", rocExprFilters);
```

Comment:
Can we use the flag name constants as the keys in this map? I'm hesitant to use copy-pasted strings that are susceptible to typos. e.g.:
```suggestion
    rocFiltersTypeMap.put(ROC_EXPR, rocExprFilters);
```

Reply:
This is a good point. I updated it for `ROC_EXPR` and `ROC_REGIONS`, but unsure what to do about `rocGenotypeFilters` and `rocVariantFilters` since they didn't have analogous constants.